packages/react/scroll-area/src/ScrollArea.tsx

```diff
@@ -301,7 +303,7 @@ const ScrollAreaNative = forwardRefWithAs<typeof ROOT_DEFAULT_TAG, ScrollAreaNat
 );
 
 type ScrollAreaImplProps = ScrollAreaInternalProps &
-  Omit<ScrollAreaOwnProps, 'unstable_forceNative'>;
+  Omit<ScrollAreaOwnProps, 'unstable_forceNative' | 'selector'>;
```

Comment:
Why are we omitting `selector` here? Isn't this the non-native scrollarea?

Reply:
We shouldn't be lol... will add it back in.